spec/requests/items_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe 'Items API', type: :request do
+  it 'returns a successful response when there are no items' do
+
+    get '/api/v1/items'
+
+    expect(response).to have_http_status(:success)
+    expect(JSON.parse(response.body).size).to eq(0)
+  end
+  it 'returns a single item' do
+    FactoryBot.create(:item, name: 'Raspberry', sell_in: 5, quality: 10)
+
+    get '/api/v1/items'
+
+    expect(response).to have_http_status(:success)
+    expect(JSON.parse(response.body).size).to eq(1)
```

Comment:
Comment for future PR, no change needed - You have the advantage of working with JSON here instead of HTML. JSON is really easy to test. So you could specify something like:

`expect(JSON.parse(response.body).fetch("name")).to eq("Raspberry")`